scripts/env/prod.sh

```
@@ -0,0 +1,347 @@
+# Performance validation
+performance_check() {
+    local base_url="${1:-http://localhost:3001}"
+    
+    log_info "Running performance checks against $base_url"
+    
+    # Check response times
+    log_info "Checking response times..."
+    local response_time=$(curl -o /dev/null -s -w '%{time_total}\n' "$base_url")
+    
+    # Convert to milliseconds for easier comparison
+    local response_ms=$(echo "$response_time * 1000" | bc -l | cut -d. -f1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)